news/2018-09-19-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "The QAT Gazette: September 2018"
+date: 2018-10-19 16:00:00 +0000
+---
+
+Huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and a brand new QAT Gazette contributor? Another month's worth of exciting news beckons, so read on to catch up on all the excitement!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+A new month brings new adventure, new challenges, and a new face to our humble blog!
+
+A lot has happened in the past month and as you might expect, you can find it all right here! From substantial changes to the way Beatmap Nominator applications are conducted, to ongoing developments within the Modding Discussions system and of course, some aforementioned new faces, keep reading to find out everything you need to know in and around the mapping and modding community.
```

Comment:
has happened -> happened

Reply:
"has happened" is perfect tense which implies the things have been completed - which as the article later puts forward, is the case, hence why I used it.

Also it just reads better imo.